CefSharp.OutOfProcess.BrowserProcess/BrowserProcessHandler.cs

```diff
@@ -124,5 +134,48 @@ void IOutOfProcessClientRpc.SetFocus(int browserId, bool focus)
 
             browser?.GetBrowserHost().SetFocus(focus);
         }
+
+        /// <summary>
+        /// Update Request Context Preferences of the browser.
+        /// </summary>
+        /// <param name="browserId">The browser id.</param>
+        /// <param name="preferences">The preferences.</param>
+        void IOutOfProcessClientRpc.UpdateRequestContextPreferences(int browserId, IDictionary<string, object> preferences)
+        {
+            var browser = _browsers.FirstOrDefault(x => x.Id == browserId);
+
+            CefThread.ExecuteOnUiThread(() =>
+            {
+                if (browser?.GetRequestContext() is IRequestContext requestContext)
+                {
+                    foreach (KeyValuePair<string, object> pref in preferences)
+                    {
+                        requestContext?.SetPreference(pref.Key, pref.Value, out _);
```

Comment:
There's no point looping through all the kvps in the dict if the request context is null.

Reply:
i removed the null-conditional operator, because the "is"-clause makes sure we only proceed with a non-null requestContext